src/conf.R

```diff
@@ -177,6 +179,7 @@ AddSetting = function(name,  comment, default=0, unit="1", adjoint=F, derived, e
 	} else {
 		if (missing(equation)) stop("'derived' provided, but no 'equation' in AddSetting")
 	}
+	if (name == 'Ave') Ave <<-default; 
```

Comment:
You don't need to go through here to modify a global variable.

You can just have a "Ave = TRUE" somewhere in the beginning of Dynamics.R